scipy/stats/_distn_infrastructure.py

```diff
@@ -2212,21 +2239,36 @@ def fit(self, data, *args, **kwds):
               function to be optimized) and ``disp=0`` to suppress
               output as keyword arguments.
 
+            - method : The method to use. The default is "MLE" (Maximum
+              Likelihood Estimate); "GMM" (Generalized Method of Moments)
```

Comment:
if the strings -> (`'mm', 'mle') are going to be lower case, maybe make these in the docstring lower case?

Reply:
These are acronyms, which are traditionally capitalized, so I'd like to keep them that way to be more familiar to the user.

Reply:
ah, I see now. The conversion happens when checking against the keys in the method dict, makes sense.